css/style.css

```diff
@@ -24,6 +24,27 @@ a.btn:hover
     background-color: lightgray;
 }
 
+/* replaces <big> tag */
+em.big
+{
+    font-style: normal;
+    font-size: larger;
+}
+
+/* replaces <tt> tag */
+em.tt
+{
+    font-style: normal;
+    font-family: monospace;
+}
+
+/* replaces <u> tag */
+em.u
```

Comment:
It's semantic was changed and it should no longer be used to underline text.
http://www.w3.org/TR/html5-diff/#changed-elements